src/assert.rs

```diff
@@ -125,6 +130,25 @@ impl Assert {
         self
     }
 
+    /// Sets environments variables for the command.
+    ///
+    /// # Examples
+    ///
+    /// ```rust
+    /// extern crate assert_cli;
+    ///
+    /// assert_cli::Assert::command(&["printenv"])
+    ///     .with_env(&[("TEST_ENV", "OK")])
+    ///     .stdout().contains("TEST_ENV=OK")
+    ///     .execute()
+    ///     .unwrap();
+    /// ```
```

Comment:
Feel free to add some more examples here as they are the real integration tests :)